bootstrap/bootstrap.ps1

```diff
@@ -67,7 +67,10 @@ if (-Not (Test-Path $junctionParent))
 {
     mkdir $junctionParent | Out-Null
 }
-cmd.exe /c mklink /j $SharedFrameworkSymlinkPath $junctionTarget | Out-Null
+if (-Not (Test-Path $SharedFrameworkSymlinkPath))
```

Comment:
Peter actually ran into this case (he was updating .cliversion in a client repo), I logged https://github.com/dotnet/buildtools/issues/1192 to take care of it.